### PR TITLE
implemented user kick

### PR DESCRIPTION
**Pull request by Totenfluch**

### CODE REVIEW

File: screens/chats/ChatScreen.js [hunk 1]
```diff
@@ -99,9 +99,12 @@ export default class ChatScreen extends React.Component {
     } 
 
     updateParticipants(chat){
```
Comment (Totenfluch) [inline screens/chats/ChatScreen.js]: @lukasHoel wieso wird diese Methode alle ~5s aufgerufen?

**Comment by Totenfluch** [comment]
wait for merge-ready

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: screens/chats/ChatScreen.js (hunk 1 above)
Comment (lukasHoel) [reply screens/chats/ChatScreen.js]: Checkt alle 5 Sekunden, ob der User noch online ist und ändert ggf die Anzeige

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by Totenfluch** [comment]
merge-ready

**Comment by Totenfluch** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@hschat/hschat?release-channel=kick-users

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Pull request merged**